frontend/src/modules/Traces/LatencyModalForm.tsx

```diff
@@ -13,13 +13,22 @@ const LatencyModalForm: React.FC<LatencyModalFormProps> = ({
 	onCancel,
 	latencyFilterValues,
 }) => {
-	const [form] = Form.useForm();
+    const [form] = Form.useForm();
+    const [disabled, setDisabled] = useState<boolean>(false)
+    const [min, setMin] = useState<number>(parseInt(latencyFilterValues.min))
+    const [max, setMax] = useState<number>(parseInt(latencyFilterValues.max))
```

Comment:
Instead of creating state variables is it possible to do it with `form.getFieldValue("name-of-field")` ?

Reply:
Oh.., but how can we update `min` & `max`, without a state variable ?

Reply:
Yeah you are right, I just confirmed `form.getFieldValue()` doesn't return updated values since it doesn't re-render the component. 

You can just use the validator rules in `Form.Item` this will remove state variables, and custom error message since error handling would be handled by Antd. Ref: https://ant.design/components/form/#Custom-validator-not-working, https://ant.design/components/form/#Rule. I guess this would be the ideal solution. What do you think? @ankitnayan

Reply:
@jyash97 correct, using antd would be better.